subsys/net/l2/ethernet/ethernet.c

```diff
@@ -288,6 +288,7 @@ static enum net_verdict ethernet_recv(struct net_if *iface,
 		}
```

Comment:
Please add commit message to the priority commit as zephyr requires it.

Reply:
@go2sh ping :)